src/QuantumSdk/QuantumSdk.nuspec

```diff
@@ -24,5 +24,6 @@
     <file src="ProjectSystem\**\*" target="ProjectSystem" exclude="**\*.v.template"/>
     <file src="Tools\BuildConfiguration\bin\$Configuration$\netcoreapp3.1\publish\**" target="tools\utils" exclude="**\*.pdb" />
     <file src="..\QsCompiler\CommandLineTool\bin\$Configuration$\netcoreapp3.1\publish\**" target="tools\qsc" exclude="**\*.pdb" />
+    <file src="MS_Quantum_Spot_Dev.png" target="images" />
```

Comment:
would it be better to have it on a single location, to avoid having the same file duplicated?

Reply:
I considered it. However, following the example of the extensions, and since the file is small, I opted for keeping a copy together with each package so they are self contained, and also so we could:
 1) Update them independently if needed.
 2) Avoid the dependency on the path or an external folder in case they get moved.
I'm happy to update this to a single location, if you think it would be better....

Reply:
Per our conversation, I'm moving this to a shared location that will be common across repos.